libs/ui/src/components/fields/input-text-field/InputTextField.tsx

```diff
@@ -19,6 +20,7 @@ export function InputTextField({
   label,
   variant,
   required = false,
+  helperTextMessage = true,
```

Comment:
i'd put `helperTextMessage` to false, since we mainly use it to display errors, if someone wants to show something else they'll need to pass a `helperText` prop to override the current functionality